src/binary/uint.rs

```diff
@@ -73,6 +69,45 @@ impl UInt {
     }
 }
 
+/// The result of the [`magnitude_to_bytes`] function. The fields are private;
+/// users must go through `as_ref()` to see the slice that ignores leading
+/// zeros.
```

Comment:
This doc comment reads as a bit too "how".  Maybe:

```suggestion
/// The big-endian, compact slice of bytes for a `u64`.  Leading zero octets are
/// not part of the representation.
```